src/Microsoft.Health.Fhir.CodeGen.Tests/CrossVersionTests.cs

```diff
@@ -20,33 +20,41 @@ namespace Microsoft.Health.Fhir.CodeGen.Tests;
 
 public class CrossVersionTests
 {
-    [Theory]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R2toR3", "2to3")]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R3toR2", "3to2")]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R3toR4", "3to4")]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R4toR3", "4to3")]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R4toR5", "4to5")]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R5toR4", "5to4")]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R4BtoR5", "4Bto5")]
-    [InlineData("C:\\git\\fhir-cross-version\\input\\R5toR4B", "5to4B")]
+    public CrossVersionTests(ITestOutputHelper outputWriter)
+    {
+        Console.SetOut(new TestWriter(outputWriter));
+    }
+
+    [Theory(DisplayName = "TestLoadingFml")]
+    [InlineData("R2toR3", "2to3")]
+    [InlineData("R3toR2", "3to2")]
+    [InlineData("R3toR4", "3to4")]
+    [InlineData("R4toR3", "4to3")]
+    [InlineData("R4toR5", "4to5")]
+    [InlineData("R5toR4", "5to4")]
+    [InlineData("R4BtoR5", "4Bto5")]
+    [InlineData("R5toR4B", "5to4B")]
     public void TestLoadingFml(string path, string versionToVersion)
     {
+        string prefixPath = @"C:\git\fhir-cross-version\input\";
```

Comment:
We cannot actually hardcode the paths here - I had done that in a rush pre WGM, but these tests cannot run on all platforms.  I will update post-merge, but wanted to note here.